### PR TITLE
[TACHYON-793] fixing deploy script for virtualbox 

### PR DESCRIPTION
**Pull request by yuzhu**
https://tachyon.atlassian.net/browse/TACHYON-793

Somehow the combination of centos6.5 + java7 + maven 3.3.3 did not work.. 
so this is an attempt to make local vb-based vagrant scripts work again.

### CODE REVIEW

File: deploy/vagrant/provision/roles/common/files/java.sh [hunk 1]
```diff
@@ -4,5 +4,8 @@
 
 # install java
 sudo yum install -y -q java-1.7.0-openjdk-devel.x86_64
-jvm="/usr/lib/jvm/java-1.7.0-openjdk.x86_64"
+
+jbinary=`realpath /etc/alternatives/java`
+jvm=$(dirname $jbinary)/..
```
Comment (calvinjia) [inline deploy/vagrant/provision/roles/common/files/java.sh]: Does this work on the other images like Amazon's linux?

Comment (yuzhu) [inline deploy/vagrant/provision/roles/common/files/java.sh]: yes, it does.

**Comment by uronce-cc** [comment]
+1 LGTM

**Comment by calvinjia** [comment]
+1 LGTM

**Pull request merged**